platform/darwin/src/MGLRasterSource.h

```diff
@@ -104,6 +104,109 @@ MGL_EXPORT
  */
 - (instancetype)initWithIdentifier:(NSString *)identifier configurationURL:(NSURL *)configurationURL tileSize:(CGFloat)tileSize NS_DESIGNATED_INITIALIZER;
 
+/**
+ Returns a raster source initialized an identifier, tile URL templates, and
+ options.
+
+ After initializing and configuring the source, add it to a map view’s style
+ using the `-[MGLStyle addSource:]` method.
+
+ #### Tile URL templates
```

Comment:
There’s the line `@interface MGLVectorSource : MGLTileSource` at the top, which links to the documentation for [MGLTileSource](https://www.mapbox.com/ios-sdk/api/3.5.0/Classes/MGLTileSource.html). However, that page says “Do not create instances of this class directly”, which the developer may misinterpret to mean “don’t use these methods at all”. This is a shortcoming of our documentation for abstract classes in general (and of some of Apple’s own abstract class documentation).